Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -45,6 +45,8 @@ import BraintreeDataCollector
     /// - Parameter apiClient: The API Client
     @objc(initWithAPIClient:)
     public init(apiClient: BTAPIClient) {
+        apiClient.shouldSendAPIRequestLatency = true
```

Comment:
qq: do we always want this to be set to true? 👀

Reply:
For PayPal currently yes we want to send this for all `BTPayPalClient` flows so we can track this across both current and future QL initiatives. This branch will get merged into our other feature branches to ensure we are tracking these events there as well.